src/octopeer-github/main/URLHandler.ts

```diff
@@ -50,4 +50,14 @@ abstract class URLHandler {
         }
     }
 
+    /**
+     * Checks whether the url provided is one that belongs to a Pull Request.
+     * 'Correct' format:  http[s]//[...]github.com/(owner)/(repo)/pull/(pr-no)[/...]
+     * @param url               The URL to get the session ID from.
+     * @returns {boolean} false if the url is INVALID. true if the url is VALID.
+     */
+    public static isPullRequestUrl(url: string): boolean {
```

Comment:
Is this method correctly tested? Because I do not see any added tests for this method. This might be useful for regression tests.